source/reviews/comments.txt

```diff
@@ -0,0 +1,112 @@
+==========================================
+How To Write Code And Copy Review Comments
+==========================================
+
+Summary
+-------
+
+- Be kind.
+- Explain your reasoning.
+- Balance giving explicit directions with just pointing out problems and letting
+  the author decide.
+- Encourage authors to simplify code or add code comments instead
+  of just explaining the complexity to you.
+- Encourage authors to simplify explanations when appropriate. We're all
+  smart, but smart words take more time to read. Readers prefer
+  conciseness and the simplest explanation in most situations.
+
+Label Your Comments
++++++++++++++++++++
+
+Label comments similar to `Conventional Comments <https://conventionalcomments.org/>`__
+so the author understands this is a suggestion, issue, question, or comment.
+
+- **No "nits"**.
```

Comment:
This surprises me because I label little improvements "nit" all the time - I think of it as "optional, maybe a style issue, but I do think this would be better - change if you agree".

It sounds like these would just be "suggestions," but I think it can be helpful to convey the weight of the suggestion sometimes.

Just thinking out loud, this system is fine with me.